C++/3. Operators.md

```diff
@@ -2,42 +2,38 @@
 
 ## Start
 
-There are certain operators which do addition, and things like that in C++. In this file, I will teach you the basic operators:
+C++ provides certain operators which do addition, and things like that in C++. In this file, I will teach you the basic arithmetic operators:
```

Comment:
This doesn't make sense. It should be, 'C++ provides certain operators which do addition, and things like that.

Reply:
I felt like the wording was off, and now I see it, do you have any suggested wording to use instead?